components/navigation.tsx

```diff
@@ -31,189 +32,255 @@ export default function Navigation() {
+                      </Dropdown.Item>
+                      <Dropdown.Item>
+                        <Link href="/solutions/marketers">
+                          <a className="nav-link" role="button">
+                            Marketing
+                          </a>
+                        </Link>
+                      </Dropdown.Item>
+                      <Dropdown.Divider />
+                      <Dropdown.Header>
+                        <FontAwesomeIcon
+                          color="black"
+                          icon={"check"}
+                          size="1x"
+                          className="pr-1"
+                        />
+                        Categories
+                      </Dropdown.Header>
+                      <Dropdown.Item>
+                        <Link href="/solutions/reverse-etl">
+                          <a className="nav-link" role="button">
+                            Reverse ETL
+                          </a>
+                        </Link>
+                      </Dropdown.Item>
+                      <Dropdown.Item>
+                        <Link href="/solutions/customer-data-platform">
+                          <a className="nav-link text-wrap" role="button">
+                            Customer Data Platform
+                          </a>
+                        </Link>
+                      </Dropdown.Item>
+                    </Col>
+                  </Row>
+                </Container>
+              </NavDropdown>
 
-              <Nav.Link className="d-md-none d-lg-block">
-                <div style={{ paddingTop: 4 }}>
-                  <iframe
-                    src="https://ghbtns.com/github-btn.html?user=grouparoo&repo=grouparoo&type=star&count=false"
-                    frameBorder="0"
-                    scrolling="0"
-                    width="65"
-                    height="20"
-                    title="GitHub"
-                  ></iframe>
-                </div>
+              <Nav.Link>
+                <Link href="/pricing">
+                  <a className="nav-link" role="button">
+                    Pricing
+                  </a>
+                </Link>
               </Nav.Link>
 
-              <Button
-                variant="primary"
-                href="/docs/getting-started"
-                size="sm"
-                style={{ marginTop: 7, marginBottom: 20 }}
-              >
-                Get Started
-              </Button>
+              <Nav.Link>
+                <Link href="/blog">
+                  <a className="nav-link" role="button">
+                    Blog
+                  </a>
+                </Link>
+              </Nav.Link>
             </Nav>
+            <Button
+              variant="outline-dark"
+              href="https://github.com/grouparoo/grouparoo"
+              size="sm"
+              className="col-sm-12 d-md-none d-lg-inline-block col-lg-1 mx-lg-1 mx-0 mt-3"
+            >
```

Comment:
I believe we still want this link to open in a new tab (`target="_blank"`)

Reply:
Thanks for catching that one!